docs/components/connectors/out-of-the-box-connectors/salesforce.md

```diff
@@ -0,0 +1,128 @@
+---
+id: salesforce
+title: Salesforce Connector
+description: Manage your Salesforce Instance from your BPMN process. Learn how to create a Salesforce Connector task, and get started.
+---
+
+The Salesforce Connector is an outbound protocol Connector that allows you to connect your BPMN service with [Salesforce](https://salesforce.com/) to interact with the [Salesforce APIs](https://developer.salesforce.com/docs/apis).
+
+## Prerequisites
+
+To use the **Salesforce Connector**, you must have a [Salesforce Connected App with OAuth 2.0 Client Credentials Flow](https://help.salesforce.com/s/articleView?id=sf.connected_app_client_credentials_setup.htm&type=5).
+
+:::note
+It is highly recommended not to expose your Salesforce Connected App Client Id and Client Secret as plain text. Instead, use Camunda secrets. Learn more in our documentation on [managing secrets](/components/console/manage-clusters/manage-secrets.md).
```

Comment:
```suggestion
It is highly recommended not to expose your Salesforce connected application client ID and client secret as plain text. Instead, use Camunda secrets. Learn more in our documentation on [managing secrets](/components/console/manage-clusters/manage-secrets.md).
```

Reply:
Salesforce Connected App is the name of a product. I made it _italic_ to show this.